kubernetes_tests/test_base.py

```diff
@@ -264,8 +266,11 @@ def start_dag(self, dag_id, host):
         assert result.status_code == 200, f"Could not enable DAG: {result_json}"
         post_string = f"http://{host}/api/v1/dags/{dag_id}/dagRuns"
         print(f"Calling [start_dag]#2 {post_string}")
+
         # Trigger a new dagrun
-        result = self.session.post(post_string, json={})
+        result = self.session.post(
+            post_string, json={"logical_date": timezone.coerce_datetime(timezone.utcnow())}
```

Comment:
I think this can simply use `datetime.datetime.now(datetime.utc)` like in Docker tests. Don’t need to import Airflow here.

Reply:
Also we need to covert this to a string.

Reply:
I have changed it.